src/radiation/radiation.cpp

```diff
@@ -130,7 +132,7 @@ void ablate::radiation::Radiation::Setup(const solver::Range& cellRange, ablate:
                 virtualcoord[ipart].z = centroid[2] + (virtualcoord[ipart].zdir * 0.1 * minCellRadius);
 
                 /** Update the physical coordinate field so that the real particle location can be updated. */
-                UpdateCoordinates(ipart, virtualcoord, coord);
+                UpdateCoordinates(ipart, virtualcoord, coord, 0.0);  //! adv value of 0.0 places the particle exactly where the virtual coordinates are.
```

Comment:
I consolidated the UpdateCoordinates function to take an "advancement" as a parameter. This pushes the DMSwarm coordinates ahead of the virtual coordinate by some factor of the minimum cell radius. This becomes useful later.